src/Create.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+// we will redirecte the user after finish adding the blog to another page using hook
+import { useHistory } from 'react-router-dom';
+
+const Create = () => {
+    const [title,setTitle] = useState('');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/7bel6/b-blog/pullRequest?prid=7919486)
```suggestion
    const [title,setTitle] = useState("");
```